frontend/src/components/folderpage/CreateNote.js

```diff
@@ -1,9 +1,60 @@
 import React, { useState } from "react";
 import { Link } from "react-router-dom";
 import FileSettings from "../shared/FileSettings";
+import axios from 'axios'
 
 function CreateNote(props) {
     const [name, setName] = useState("");
+
+    const [color, setColor] = useState("");
+
+    const [isPrivate, setIsPrivate] = useState(false);
+
+    const [passwordA, setPasswordA] = useState("");
+    const [passwordB, setPasswordB] = useState("");
+
+    function submitNoteName() {
+        props.setNoteName(name);
+        props.setShowNoteModal(false);
+        const note = {
+            name: name,
+            color: color,
+            isPrivate: isPrivate,
+            password: passwordA,
+            contents: [],
+            isLocked: false
```

Comment:
I'm wondering what is this isLocked used for since there's a isPrivate already.

Reply:
It sounds like we were planning on refactoring that later after conversation with christine